module_utils/vcd.py

```diff
@@ -43,7 +44,15 @@ def login(self):
                                  api_version=api_version,
                                  verify_ssl_certs=verify_ssl_certs)
 
-            self.client.set_credentials(BasicLoginCredentials(user, org, password))
+            if user == 'API_TOKEN':
+                oAuthResponse = post(
```

Comment:
might want to pass verify=verify_ssl_certs as option to post() so that in case someone says verify_ssl_certs: False in their tasks it would do as it is supposed to